src/Mvc/Mvc.ViewFeatures/src/Buffers/PagedBufferedTextWriter.cs

```diff
@@ -111,7 +111,7 @@ public override Task WriteAsync(char value)
 
             // FlushAsyncCore will return CompletedTask if nothing sync buffered
```

Comment:
Probably should delete these comments or update to something like
```
// If nothing sync buffered or only a single operation is already completed,
// flushTask.IsCompletedSuccessfully will be true, so fast-path and skip async state-machine
```

Same on https://github.com/aspnet/AspNetCore/pull/11606/files#diff-d6564431f116b915d3f4b3b4cb508867R129 and https://github.com/aspnet/AspNetCore/pull/11606/files#diff-d6564431f116b915d3f4b3b4cb508867R146

Reply:
Removed